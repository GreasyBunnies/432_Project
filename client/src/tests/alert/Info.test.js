import React from 'react';
import ReactDOM from 'react-dom';
import Info from '../../components/alert/Info.js';

import 'jest-localstorage-mock';
import renderer from 'react-test-renderer';

it('renders Info without crashing', () => {
	const div = document.createElement('div');

	ReactDOM.render(<Info />, div);
	ReactDOM.unmountComponentAtNode(div);
});

it('displays the correct message', () => {
	const component = renderer.create(
		<Info header="Test" message="This is a test"></Info>,
	);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});