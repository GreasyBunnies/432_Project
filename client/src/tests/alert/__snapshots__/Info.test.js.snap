// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays the correct message 1`] = `
<div
  className="fade alert alert-info show"
  role="alert"
>
  <div
    className="alert-heading h4"
  >
    Test
  </div>
  This is a test
</div>
`;
